plugins/chat/lib/chat/notifier.rb

```diff
@@ -127,6 +125,8 @@ def list_users_to_notify
       [to_notify, inaccessible, all_mentioned_user_ids]
     end
 
+    private
+
     def expand_global_mention(to_notify, already_covered_ids)
```

Comment:
Should we do this in `expand_here_mention` as well? Unless bots already cannot be counted as "here"

Reply:
hmm yes that's a good point, will check